### ISSUE TITLE
index.html in root renders as empty page

### ISSUE THREAD

**Issue by bodigital** [tighten/jigsaw#547]
Hi,
I experimented with collections and tried to map one collection to the root by changing the path of the collection to '/' in config.php. I soon realized that while it worked, it's not wise and reverted to changes in config.php. I pasted the current contents below.
My issue is, that index.html in root now renders as an empty page. I tried renaming index.blade.php it rendered just fine. Once I changed it back to index.blade.php it rendered an empty page again... 
Any ideas?

config.php

<?php
use Illuminate\Support\Str;

return [
    'production' => false,
    'baseUrl' => '',
    'siteName' => 'Testflight',
    'collections' => [
        'people' => [
            'path' => 'people',
            'sort' => 'order',
        ],
        'objects' => [
            'path' => 'objects',
        ],
    ],
    'selected' => function ($page, $section) {
        return Str::contains($page->getPath(), $section) ? 'selected' : '';
    },
];

**Comment by bakerkretzmar** [comment]
Can you share the complete contents of `config.php` (the beginning was cut off in the snippet you shared), and also the contents of the `index.blade.php` file you're trying to render?

**Comment by bodigital** [comment]
sorry, I should have attached the files. Here you go:
[index.blade.txt](https://github.com/tighten/jigsaw/files/6073285/index.blade.txt)
[config.txt](https://github.com/tighten/jigsaw/files/6073279/config.txt)

**Comment by bakerkretzmar** [comment]
Thanks, can you share `_layouts/master.blade.php` too?

**Comment by bodigital** [comment]
[master.blade.txt](https://github.com/tighten/jigsaw/files/6073821/master.blade.txt)

**Comment by bakerkretzmar** [comment]
Do you get any errors in the console running `vendor/bin/jigsaw build`? Or could it be something in the header or footer partials? When I drop those files into a new Jigsaw site locally it builds fine.

**Comment by bodigital** [comment]
Sorry for the late reply. Good news: I found the issue. 
The file extensions in the the folder of one of the collections were **.blade.php**. Once I changed these to .md the index page in the root folder rendered fine again. 

I was able to reproduce the issue in  a complete fresh Jigsaw site by:
- defining a collection in config.php
- creating the folder for the collection and adding an empty file test.blade.php

This resulted in the rendered index.html to be empty. 
Adding a file hello.blade.php in the source folder rendered /hello/index.html just fine.

All builds went without any errors.
